v0/destinations/google_ads_enhanced_conversion/util.js

```diff
@@ -0,0 +1,148 @@
+const { get, set } = require("lodash");
+const sha256 = require("sha256");
+const { httpSend } = require("../../../adapters/network");
+const { isHttpStatusSuccess } = require("../../util/index");
+const { TRANSFORMER_METRIC } = require("../../util/constant");
+const ErrorBuilder = require("../../util/error");
+const {
+  DISABLE_DEST,
+  REFRESH_TOKEN
+} = require("../../../adapters/networkhandler/authConstants");
+const Cache = require("../../util/cache");
+
+const conversionActionIdCache = new Cache(24 * 60 * 60);
```

Comment:
We should store the cache time in constant and source it here

Reply:
Made that a constant and provided scope for getting from .env also.